src/fonts.rs

```diff
@@ -8,7 +8,7 @@ pub fn init(font_dirs: &[String], font_family: &str) -> Option<(fontdb::Database
 
     let mut families = font_family
         .split(',')
-        .map(|name| name.trim())
+        .map(str::trim)
```

Comment:
This shouldn't result in any user-facing changes.

The goal here is to improve readability.

Reply:
I'd argue both are equally readable, just the without closure is shorter. But I like it :)